device/core/devdoc/device_client_requirements.md

```diff
@@ -91,19 +91,27 @@ client.sendEvent(new Message('hello world'), print);
 **SRS_NODE_DEVICE_CLIENT_16_058: [** The `close` method shall immediately call the `closeCallback` function if provided and the transport is already disconnected. **]**
 
 #### sendEvent(message, sendEventCallback)
+#### sendEvent(outputName, message, sendEventCallback)
```

Comment:
>sendEvent [](start = 5, length = 9)

if we must have a special sendEvent method (instead of using the message property) I wonder if we should use a different method name since overloading in javascript/typescript can be a pain.
sendEventTo or sendTo for example (i'm not a fan of the Event terminology.. we talk about telemetry, d2c... we rarely use event anywhere I think. would have to double check with other SDKs.